### DISCUSSION TITLE
dontReorderAssociatables() on BelongsToMany ( for Scout searchable models )

### DISCUSSION THREAD

**Discussion by olivM** [laravel/nova-issues#4742] [Ideas & Feature Requests] [answered]
Can we get some kind of dontReorderAssociatables() for BelongsToMany ?

as the AttachableController do a
```
->sortBy('display', SORT_NATURAL | SORT_FLAG_CASE)
```
and by so, it mess up the order returned by Scout search

thanks !

**Comment by crynobone** [comment, marked as answer]
Implemented in 4.15.0

This will be applied automatically when Resource uses Scout, or can be activated via:

```php
BelongsToMany::dontReorderAttachables(),
```
